manager/integration/tests/test_basic.py

```diff
@@ -4588,21 +4593,34 @@ def test_backup_volume_restore_with_access_mode(set_random_backupstore, # NOQA
        => Validate the access mode should be the same as the specified
     """
     # Step 1
-    test_volume_name = generate_volume_name()
+    test_volume_name = "test-volume-" + str(random.randint(0, 9))
     client.create_volume(name=test_volume_name,
                          size=str(DEFAULT_VOLUME_SIZE * Gi),
                          numberOfReplicas=2,
                          accessMode=access_mode)
     wait_for_volume_creation(client, test_volume_name)
     volume = wait_for_volume_detached(client, test_volume_name)
-    volume.attach(hostId=common.get_self_host_id())
-    volume = common.wait_for_volume_healthy(client, test_volume_name)
+
+    # Since RWX volumes can only be manually attached in maintenance mode,
+    #  the volume was created by PV and PVC deployed.
```

Comment:
# Since RWX volumes can only be manually attached in maintenance mode, 
    # the volume should be created by PVC and PV instead to ensure the volume attach/deattach successfully.